src/Util/Semaphore/UnixSemaphore.php

```diff
@@ -13,7 +14,7 @@ class UnixSemaphore implements Semaphore {
 	private $semaphoreKey;
 	/** @var int */
 	private $capacity;
-	/** @var resource|null the lazily initialized semaphore descriptor */
+	/** @var resource|SysvSemaphore|null the lazily initialized semaphore descriptor */
```

Comment:
genuine question: do we still need `resource` now that we have `SysvSemaphore`?

(same in other places)

Reply:
Good point, no it seems not. Removed!